features/board/src/main/java/com/techcafe/todone/board/CreateBoardFragment.kt

```diff
@@ -0,0 +1,47 @@
+package com.techcafe.todone.board
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.content.DialogInterface
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.observe
+import com.techcafe.todone.board.databinding.FragmentCreateBinding
+import com.techcafe.todone.board.viewModel.BoardCreateViewModel
+
+class CreateBoardFragment : DialogFragment() {
+    private lateinit var dialog: AlertDialog
+    private lateinit var binding: FragmentCreateBinding
+    private val viewmodel: BoardCreateViewModel by viewModels()
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val inflater = requireActivity().layoutInflater
+        binding = FragmentCreateBinding.inflate(inflater)
+        binding.viewModel = viewmodel
+        binding.lifecycleOwner = this
+
+        dialog = activity?.let {
+            val builder = AlertDialog.Builder(it)
+            builder.setView(binding.root)
+                .setPositiveButton("作成", null)
+                .setNegativeButton("キャンセル",
```

Comment:
緑の波線とか、灰色になってる場合はIDEがもっと綺麗にかけるよって教えてくれてるサイン！
```suggestion
.setNegativeButton("キャンセル") { dialog, _ ->
                    dialog.cancel()
}
```

こう書けるっぽいね〜